blog.html

```diff
@@ -0,0 +1,103 @@
+og_url: https://mailmeteor.com/blog
+og_image: https://mailmeteor.com/assets/img/opengraph/blog-mail-merge-gmail.jpg
+---
+
+<!DOCTYPE html>
+<html lang="{{ page.lang | default: site.lang | default: 'en' }}">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0" />
+    <meta name="google-site-verification" content="TShcwwV_esZ98tyJWg6fRMCTDcjUwGnfGBu087vga3M" />
+    <link rel="icon" href="/assets/img/brand/favicon.ico" type="image/x-icon" />
+
+    <title>{{ page.title }}</title>
+    <link rel="canonical" href="{{ page.canonical }}" />
+    <meta name="description" content="{{ page.description }}" />
+
+    <!-- Open graph -->
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="{{ page.og_title }}" />
+    <meta property="og:description" content="{{ page.og_description }}" />
+    <meta
+      name="image"
+      property="og:image"
+      content="{{ page.og_image | default: 'https://mailmeteor.com/assets/img/opengraph/main.jpg' }}"
+    />
+    <meta property="og:url" content="{{ page.og_url }}" />
+    <meta name="twitter:site" content="@mailmeteor" />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:url" content="{{ page.og_url }}" />
+    <meta name="twitter:title" content="{{ page.og_title }}" />
+    <meta name="twitter:description" content="{{ page.og_description }}" />
+    <meta
+      name="twitter:image"
+      content="{{ page.og_image | default: 'https://mailmeteor.com/assets/img/opengraph/main.jpg' }}"
+    />
+
+    <!-- Styles -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="/assets/css/main-20210313.css" />
+    <link rel="stylesheet" href="/assets/css/blog.css" />
+  </head>
+  <body class="blogpost">
+    <article itemid="{{ page.canonical }}" itemscope itemtype="https://schema.org/BlogPosting">
+      <header class="blogpost-header">
+        <div class="container">
+          <nav class="row">
+            <div class="col-xs-12 col-md-6 col-lg-8">
+              <a href="/"><img src="/assets/img/brand/logo_mailmeteor_white.svg" alt="Mailmeteor logo" id="logo" /></a>
+            </div>
+            <div class="col-md-6 col-lg-4 blogpost-header-tagline d-none d-md-block">
+              Brought to you by <a href="/">Mailmeteor</a>, the best rated mail merge tool for Gmail.
+            </div>
+          </nav>
+          <div class="text-center">
+            <img itemprop="image" src="{{page.meta_image}}" class="blogpost-header-image" />
+            <h1 itemprop="headline">{{ page.title}}</h1>
```

Comment:
```suggestion
            <h1 itemprop="name">{{ page.title}}</h1>
```

Pour la homepage, on vise le nom du blog là